sklearn/preprocessing/_encoders.py

```diff
@@ -491,12 +532,28 @@ def fit(self, X, y=None):
         else:
             self._fit(X, handle_unknown=self.handle_unknown)
             self.drop_idx_ = self._compute_drop_idx()
+
+            # check if user wants to manually drop a feature that is
+            # infrequent: this is not allowed
```

Comment:
Why not?

Reply:
it doesn't seem natural to drop some features that are infrequent, and not drop some.

it would also make the code even more complex.